src/LSST/M1M3/SS/FPGA/SimulatedFPGA.h

```diff
@@ -106,7 +113,12 @@ class SimulatedFPGA : public IFPGA {
     void writeModbusCRC(std::queue<uint16_t>* response);
     uint16_t readModbus(uint16_t data);
 
+    void monitorElevation(void);
+
+    int exitThread = false;
     float rnd[RND_CNT];
+    float mountElevation = 90.;
```

Comment:
private values and methods shall be prefixed with _ according to LSST DM C++ developer guide. Actually there is a patch (WIP) to fix this in full M1M3 SS code. When adding new private values, please prefix with _ and don't use this-> in the code to access them.